tools/py/gleacf.py

```diff
@@ -145,61 +145,83 @@ def ISRA(omega, ker, y, dparam, oprefix):
 
     # adds arguments.
-    parser.add_argument("-a", "--action", choices=["conv", "deconv"], default=None, help="choose conv if you want to obtain the response of the thermostat on the vibrational density of states; choose deconv if you want to obtain the micro-canonical density of states by removing the disturbance induced by the thermostat")
-    parser.add_argument("-iipi", "--input_ipi", type=str, default=None, help="the relative path to the i-PI inputfile")
-    parser.add_argument("-ivvac", "--input_vvac", type=str, default=None, help="the relative path to the input velocity-velocity autocorrelation function")
-    parser.add_argument("-mrows", "--maximum_rows", type=int, default=-1, help="the index of the last row to be imported from INPUT_VVAC")
-    parser.add_argument("-s", "--stride", type=int, default=1, help="the stride for importing the IVVAC and computing the kernel")
-    parser.add_argument("-dt", "--timestep", type=str, default="1 atomic_unit", help="timestep associated with the vvac. <number> <unit>. Defaults to 1.0 atomic_unit")
-    parser.add_argument("-dparam", "--deconv_parameters", nargs=2, type=int, default=[500, 10], help="the parameters associated with the deconvolution. Since the operation is based on an iterative algorithm, it requires the total number of epochs NEPOCHS and the stride PSTRIDE at which the output spectrum is returned. Usage: [NEPOCHS,PSTRIDE]")
-    parser.add_argument("-oprefix", "--output_prefix", type=str, default="output", help="the prefix of the (various) output files.")
+    parser.add_argument("-A", "--action", choices=["conv", "deconv"], default=None, required=True, help="choose conv if you want to obtain the response of the thermostat on the vibrational density of states; choose deconv if you want to obtain the micro-canonical density of states by removing the disturbance induced by the thermostat")
+    parser.add_argument("-iFACF", "--input_facf", type=str, default=None, help="The input Fourier transform of the auto-correlation function (FACF). The unit of time is assumed to be atomic units unless the additional -time_scaling parameter is provided.")
+    parser.add_argument("-iXML", "--input_xml", type=str, default=None, help="The i-PI xml file that contains the Drift (A) and the Covariance (C) matrices as well as the ensemble temperature (T). The units are interpretted from the file and the A and C matrices are converted to atomic units. The C matrix is further divided by k_B T. If the -time_scaling parameter is provided, the units of A are modified using the scaling factor.")
```

Comment:
Would be good to specify somewhere that it is assumed that the acf comes from the centroid, so in case of NMGLE, the centroid would be taken. Also, you could implement that if the input contains  pile_l, it treats it like a Langevin equation. Finally, if the input contains a thermo which is none of the above, an error should be thrown.

Reply:
Added a line on nmgle.